modules/spring-config/src/main/java/org/mule/runtime/config/spring/dsl/declaration/DefaultXmlArtifactDeclarationLoader.java

```diff
@@ -108,6 +110,9 @@
  */
 public class DefaultXmlArtifactDeclarationLoader implements XmlArtifactDeclarationLoader {
 
+  public static final String SET_VARIABLE = "setVariable";
```

Comment:
Is fine for the hack, use "TRANSFORMER_" as prefix for the var name. Move TRANSFORM_IDENTIFIER out of DslConstansts and put it here too.